data/aazad20.json

```
@@ -0,0 +1,22 @@
+{
+  "name": "Sarvjeet Singh",
+  "bio": "Software Engineer",
+  "tags" : ["Software Engineer","python","react","Javascript","Tensorflow","Data Science"],
+  "socials": [
+    {
+       "icon": "FaLink",
+      "url": "https://www.linkedin.com/in/sarvjeet-singh-6249551b7/",
+      
+    }
+  ]
+  "links": [
+    {
+      "group": "portfolio",
+      "name": "My portfolio",
+      "url": "https://jeetcodes.netlify.app/",
+      "icon": "FaLink"
+    },
```

```suggestion
    }
```
The trailing comma isnot required